cilium/cmd/root.go

```diff
@@ -62,7 +62,7 @@ func init() {
 	flags.StringP("host", "H", "", "URI to server-side API")
 	viper.BindPFlags(flags)
 	rootCmd.AddCommand(newCmdCompletion(os.Stdout))
-	rootCmd.SetOut(os.Stderr)
+	rootCmd.SetOut(os.Stdout)
```

Comment:
Is this change intentional? I wonder if it might be the cause for the CI failure?

Reply:
This change is required as internally _cilium __complete_ command will be called, and it's using the stdout for showing completion options. 

I just checked the failure, seems like we are expecting help output in stderr. Let me see if there is easy way to fix this.